lib/change_stream.js

```diff
@@ -445,7 +443,7 @@ function createChangeStreamCursor(self, options) {
     }
   }
 
-  return changeStreamCursor;
+  self.cursor = changeStreamCursor;
```

Comment:
I moved this assignment inside `createChangeStreamCursor` because it should never be called and not immediately assigned to `this.cursor`. Perhaps this warrants a rename to `initializeCursor`?